tests/characters/test_bob_handles_frags.py

```diff
@@ -147,7 +147,7 @@ def test_bob_can_issue_a_work_order_to_a_specific_ursula(enacted_federated_polic
 
     # We'll test against just a single Ursula - here, we make a WorkOrder for just one.
     # We can pass any number of capsules as args; here we pass just one.
-    capsule = capsule_side_channel[0].capsule
+    capsule = capsule_side_channel()[0].capsule
```

Comment:
phew!

Reply:
Yeah, it's a little ungood there ain't it.  It's a test, though.